### PR TITLE
create and edit event screen boilerplate changes

### PR DESCRIPTION
**Pull request by d19123604**
…late changes

### CODE REVIEW

**Review by D18130435 — DISMISSED** [review]

**Comment by viniciusvieir** [comment]
Merging conflicts, @NeelSalvi could you please check?

File: client/trailseek/src/screens/CreateEventScreen.js [hunk 1]
```diff
@@ -1,177 +1,167 @@
...
-    errstl:{
-        color:"red",
-        marginBottom:10,
-    },
-
-
-const styles = StyleSheet.create({
-  maincontainer: {
-    flex: 1,
-    backgroundColor: "#ecf0f1",
```
Comment (NeelSalvi) [inline client/trailseek/src/screens/CreateEventScreen.js:160]: No ColorConstants being used to follow the colour pallet of the App

File: client/trailseek/src/screens/EditEventScreen.js [hunk 2]
```diff
@@ -13,177 +13,195 @@
...
-                />
-              )}
-              name="eventname"
-              rules={{ required: true, minLength: 4, maxLength: 15 }}
-              defaultValue=""
-            />
-            {errors.eventname && errors.eventname.type === "required" && (
-              <Text style={styles.errstl}>Event name is required.</Text>
+  const [inputs, setInput] = useState({});
+  const isAuth = useSelector((state) => state.user.isAuth);
```
Comment (NeelSalvi) [inline client/trailseek/src/screens/EditEventScreen.js:30]: Added but not used...

File: client/trailseek/src/screens/EditEventScreen.js [hunk 3]
```diff
@@ -13,177 +13,195 @@
 // const styles = StyleSheet.create({});
 
 // export default EditEventScreen;
-
 import React, { createElement, useEffect, useState } from "react";
 import { TextInput, View, StyleSheet, Image, Keyboard } from "react-native";
-import { Text, Button } from "react-native-elements";
+// import { Text, Button } from "react-native-elements";
 import { useSelector, useDispatch } from "react-redux";
 import { signIn } from "../app/userSlice";
```
Comment (NeelSalvi) [inline client/trailseek/src/screens/EditEventScreen.js:20]: Why is signin function imported here?

File: client/trailseek/src/screens/EditEventScreen.js [hunk 4]
```diff
@@ -13,177 +13,195 @@
...
+    color: "red",
+    marginBottom: 10,
+  },
+  formcontainer: {
+    backgroundColor: "#ffffff",
+    margin: 5,
+    padding: 5,
+    borderRadius: 5,
+  },
+
```
Comment (NeelSalvi) [inline client/trailseek/src/screens/EditEventScreen.js:204]: I see the Color Constants being imported but not used in the File.

**Review by NeelSalvi — COMMENTED** [review]
Please resolve these issues

**Comment by NeelSalvi** [comment]
Needs Changes and conflicts are present. The Page is going to be redone.

**Pull request closed without merging**